src/main/java/org/opentripplanner/middleware/otp/OtpRequestProcessor.java

```diff
@@ -24,7 +31,7 @@
  * is of interest the response is intercepted and processed. In all cases, the response from OTP (content and HTTP status)
  * is passed back to the requester.
  */
-public class OtpRequestProcessor {
+public class OtpRequestProcessor implements Endpoint {
```

Comment:
Should this class move to the `controllers` package? That seems like a more appropriate home for it at this point.

Reply:
Moved in https://github.com/ibi-group/otp-middleware/pull/59/commits/aee52b65a0058fc403f8e00a8c7699babd0398ed